### PR TITLE
New Badges

### PR DESCRIPTION
**Pull request by Bashamega**
Hello:)
I have integrated new badges in the readme.

### CODE REVIEW

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks great 👍 

Please could you run the command `npm run fmt` to format the files correctly

**Comment by Bashamega** [comment]
> Looks great 👍
> 
> Please could you run the command `npm run fmt` to format the files correctly

done

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 
 # HealthCheck
 
+[![RepoRater](https://repo-rater.eddiehub.org/api/badge?owner=EddieHubCommunity&name=HealthCheck)](https://repo-rater.eddiehub.org/rate?owner=EddieHubCommunity&name=HealthCheck)
+![GitHub code size in bytes](https://img.shields.io/github/languages/code-size/EddieHubCommunity/HealthCheck?style=plastic)
+![GitHub contributors](https://img.shields.io/github/contributors/EddieHubCommunity/HealthCheck)
+[![Discord](https://img.shields.io/badge/Discord-%235865F2.svg?style=plastic&logo=discord&logoColor=white)](https://discord.com/invite/jZQs6Wu)
```
Comment (eddiejaoude) [inline README.md:9]:
```suggestion
[![Discord](https://img.shields.io/badge/Discord-%235865F2.svg?style=plastic&logo=discord&logoColor=white)](http://discord.eddiehub.org)
```

**Review by eddiejaoude — APPROVED** [review]
Great! Thank you 👍

**Pull request merged**